test/python/qdmi/test_qdmi.py

```
@@ -13,35 +13,41 @@
-    """Get all available devices from a Session.
+    """Open all registered QDMI devices.
 
     Returns:
         List of all available QDMI devices.
     """
-    session = Session()
-    return session.get_devices()
+    return [open_device(device_id) for device_id in registered_device_ids()]
+
+
+def _legacy_session(**kwargs: Any) -> Session:  # ruff: ignore[any-type]
```

Ruff (0.16.1)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `**kwargs`

(ANN401)